### ISSUE TITLE
LOG

### ISSUE THREAD

**Issue by IvanM1993** [ethereum/mist#2794]
Hello. Please help with the following problem. Installed the new version 1.6.7 of get, but it references 1.6.6. 
INFO [07-19|22:13:33] Starting peer-to-peer node               instance=Geth/v1.6.6-stable-10a45cb5/windows-amd64/go1.8.3
INFO [07-19|22:13:33] Allocated cache and file handles         database=C:\\.....\\Ethereum\\chaindata cache=1024 handles=1024

**Comment by evertonfraga** [comment]
Hi, how did you install get 1.6.7? Manually or though Mist?

can you check on your `%APPDATA%\Mist\binaries` what geth version lies there? Thanks!

**Comment by IvanM1993** [comment]
Hi, evertonfraga. in the folder binaries there is a version 1.6.6. but I updated the geth to version 1.6.7 using Mist . After I relaunched the Mist gave this error. Then I manually installed version 1.6.7. But the error remained, and strangely after removal 1.6.6 binaries in a folder is automatically created the same version, after restarting the Mist. And also creates an archive of this version of the geth. What do I do next?
